src/elements/container/sticky-bar/sticky-bar.ts

```diff
@@ -61,10 +61,27 @@ export class SbbStickyBarElement extends LitElement {
   }
 
   private _toggleShadowVisibility(entry: IntersectionObserverEntry): void {
+    const isSticky = !entry.isIntersecting && entry.boundingClientRect.top > 0;
+
+    const intersectorRect = this._intersector?.getBoundingClientRect();
+    const stickyBarRect = this.shadowRoot!.querySelector(
+      '.sbb-sticky-bar__wrapper',
+    )!.getBoundingClientRect();
+    const HEIGHT_TOLERANCE = 30;
```

Comment:
Would it make sense to make this configurable/overridable? Perhaps in other scenarios the behavior might change slightly and the consumer might want to fine-tune it.

Reply:
Discussed it with Lukas, for now not